plugins/uefi-capsule/efi/meson.build

```diff
@@ -154,6 +154,11 @@ so = custom_target('fwup.so',
                              efi_ldflags + ['@INPUT@'] +
                              ['-lefi', '-lgnuefi', libgcc_file_name])
 
+# sanity check the packager set this to *SOMETHING*
+if get_option('efi_sbat_distro_id') == '' and get_option('supported_build')
```

Comment:
shouldn't they also need to set `efi_sbat_distro_generation` and the others explicitly?

Reply:
I was going to allow fallbacks to be pragmatic. e.g. falling back to `sbat_fwupd_generation` -- I also wasn't sure if `sbat_distro_pkgname` was always required; if it's a yocto build for instance there is no "package" just a blob of the finished image.

Reply:
OK..